### PR TITLE
Implementing MetaMask Portfolio Bridging

### PR DESCRIPTION
**Pull request by mapachurro**
Linea is now supported for bridging through MetaMask Portfolio.

This is going to be a powerful and convenient solution for everyday bridging purposes, greatly improving the UX of bridging between Linea and other networks.

Updating the official recommendations accordingly. The main changes are on three pages: 

/use-mainnet/bridges-of-linea
/use-mainnet/bridges-of-linea/how-to-bridge-eth
/how-to-bridge-erc20-tokens

### CODE REVIEW

File: docs/build-on-linea/tooling/data-indexers/envio.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ HyperSync is an indexed layer of the Linea blockchain, providing accelerated API
 
 - Detailed logging and error messaging are provided for effective troubleshooting and debugging.
 
-- Quickstart templates with pre-defined indexing logic for popular OpenZeppelin contracts (ERC20, ERC721, ERC1155, etc.)
+- Quickstart templates with pre-defined indexing logic for popular OpenZeppelin contracts (ERC-20, ERC-721, ERC1155, etc.)
```
Comment (jlwllmr) [inline docs/build-on-linea/tooling/data-indexers/envio.md:29]:
```suggestion
- Quickstart templates with pre-defined indexing logic for popular OpenZeppelin contracts (ERC-20, ERC-721, ERC-1155, etc.)
```

File: docs/build-on-linea/tooling/data-indexers/flair.md [hunk 2]
```diff
@@ -103,7 +103,7 @@ Explore real-world usage of Flair indexing primitives for various use-cases.
 
 ### NFT
 
-* [Index ERC721 and ERC1155 NFTs on any EVM chain with an RPC URL](https://github.com/flair-sdk/examples/tree/main/erc721-and-erc1155-nft-indexing)
+* [Index ERC721 and ERC-1155 NFTs on any EVM chain with an RPC URL](https://github.com/flair-sdk/examples/tree/main/erc721-and-erc1155-nft-indexing)
```
Comment (jlwllmr) [inline docs/build-on-linea/tooling/data-indexers/flair.md:106]:
```suggestion
* [Index ERC-721 and ERC-1155 NFTs on any EVM chain with an RPC URL](https://github.com/flair-sdk/examples/tree/main/erc721-and-erc1155-nft-indexing)
```

File: docs/use-mainnet/bridges-of-linea/index.mdx [hunk 3]
```diff
@@ -7,17 +7,40 @@ sidebar_position: 3
...
 
+
+
+### For all normal bridge traffic: 
+
+#### **We recommend you use the Bridge feature of MetaMask Portfolio: [https://portfolio.metamask.io/bridge](https://portfolio.metamask.io/bridge)**
+>
+>The Linea ecosystem has a **growing and diverse number of bridges** operating on it. 
+>
+>MetaMask Portfolio's Bridge feature **aggregates all those bridges**, and shows the options available to you so that you can **choose the provider and rate that you prefer**. Then, you can perform the bridging operation right from the dapp.
```
Comment (jlwllmr) [inline docs/use-mainnet/bridges-of-linea/index.mdx:18]: Bridge only uses a select handful of providers, which, in turn, are accessed through just two aggregators: Li.Fi and Socket. Suggest we amend to not say that Bridge uses _all_ bridges, and instead make it slightly more vague. 

see "What aggregators and providers do you use" in the FAQs [here](https://support.metamask.io/hc/en-us/articles/10055915089819-How-to-use-the-bridge#h_01GGSRF9FCZR53YCNSME5TMMY2).

File: docs/use-mainnet/bridges-of-linea/index.mdx [hunk 4]
```diff
@@ -7,17 +7,40 @@ sidebar_position: 3
...
+### For dapp teams, liquidity providers, and technology operators moving large amounts of liquidity:
+
 **Bridge ETH**
-- [Use the official Linea token bridge](/use-mainnet/bridges-of-linea/how-to-bridge-eth) (for bridging between Linea and Ethereum)
-- [Use third party bridges](#third-party-permissionless-bridges) (for bridging between Ethereum, Linea, and other networks)
 
-**Bridge ERC20 tokens**
-- [Use the official Linea token bridge](/use-mainnet/bridges-of-linea/how-to-bridge-erc20-tokens) (recommended for tech operators that are providing liquidity)
-- [Use third party bridges](#third-party-permissionless-bridges) (for bridging between Ethereum, Linea, and other networks)
+We recommend you use the official Linea token bridge:(/use-mainnet/bridges-of-linea/how-to-bridge-eth) (for bridging between Linea and Ethereum)
```
Comment (jlwllmr) [inline docs/use-mainnet/bridges-of-linea/index.mdx:27]:
```suggestion
We recommend you use the [official Linea token bridge](/use-mainnet/bridges-of-linea/how-to-bridge-eth) for bridging between Linea and Ethereum.
```

**Review by jlwllmr — CHANGES_REQUESTED** [review]
A few things to fix first

File: docs/use-mainnet/bridges-of-linea/index.mdx (hunk 3 above)
Comment (mapachurro) [reply docs/use-mainnet/bridges-of-linea/index.mdx:18]: Thank you for actually knowing the technical details....! This was me, assuming it was like Swaps.

**Pull request merged**